docs/synchronization.rst

```diff
@@ -186,3 +186,10 @@ Example::
                 await tg.spawn(use_resource, num, limiter)
 
     run(main)
+
+To adjust the number of total tokens, you can use the
+:meth:`~anyio.abc.CapacityLimiter.set_total_tokens` method.
+
+.. note:: In Trio, the total number of tokens capacity limiters is adjusted by setting the value
+    of the ``total_tokens`` property. Because waking up waiting tasks requires an ``await`` in
+    the curio backend, this cannot be supported in AnyIO.
```

Comment:
It seems to me like this note could be removed entirely. When I first read it I wondered "so is adjusting the number of tokens not supported?", but what it says is really that you need to use `.set_total_tokens()` instead of setting `.total_tokens`. I think we don't need this information, especially because understanding it requires the user to know about subtleties of trio and curio.

Reply:
Removed the comment.